.github/workflows/build.yml

```diff
@@ -18,7 +18,7 @@ jobs:
                       libretro: lib/wasm4_libretro.dylib
 
                     # glibc 2.31
```

Comment:
this comment seems stale after the change.

i guess an older version of ubuntu was preferred here for glibc compatibility reasons.
have you considered ubuntu-22.04?

Reply:
I haven't. Yeah comment might be stale. I'm away from workstation but would be worth looking at git blame for that. 22.04 is fine too, hadn't put much thought between the two except for just making it as new as possible